server/odc-service/src/main/java/com/oceanbase/odc/service/dml/converter/MySQLGeometryConverter.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.service.dml.converter;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import com.oceanbase.odc.service.dml.DataValue;
+
+import lombok.NonNull;
+
+/**
+ * @author jingtian
+ * @date 2023/11/22
+ * @since ODC_release_4.2.4
+ */
+public class MySQLGeometryConverter extends BaseDataConverter {
```

Comment:
Could you add a unit test for this in `DataConvertUtilTest`?

Reply:
None of the other DataConverters have written ut tests, so I haven’t written ut for  MySQLGeometryConverter either. I don't think it's necessary.

Reply:
DataConvertUtilTest contains all test cases for Converts

Reply:
add ut for MySQLGeometryConverter in DataConvertUtilTest